rootfs/api/settings/production.py

```diff
@@ -12,6 +12,8 @@
 # https://docs.djangoproject.com/en/1.11/ref/settings/#debug
 DEBUG = bool(os.environ.get('DEIS_DEBUG', False))
 
+LOG_LEVEL = os.environ.get('LOG_LEVEL', 'DEBUG')
```

Comment:
Default to DEBUG...? It should default to INFO as running DEBUG in production Django is not the default....

Reply:
Ok, I can see that it should be INFO. Will change it.